packages/commonwealth/client/scripts/views/components/react_quill_editor/markdown_formatted_text.tsx

```diff
@@ -193,16 +195,31 @@ export const MarkdownFormattedText = ({
 
   return (
     <>
-      <div
-        // @ts-expect-error <StrictNullChecks/>
-        ref={containerRef}
-        className={getClasses<{ collapsed?: boolean }>(
-          { collapsed: isTruncated },
-          customClass || 'MarkdownFormattedText',
-        )}
-      >
-        {finalDoc}
-      </div>
+      {isCardView ? (
+        <div className="card-view">
```

Comment:
Seems like we are just adding an extra wrapper element here. Plz move the `isCardView` logic outside of the editor components.